exchanges/request/request_test.go

```diff
@@ -174,25 +195,35 @@ func TestDoRequest(t *testing.T) {
 	t.Parallel()
 	r := New("test",
 		new(http.Client),
-		&globalshell)
+		WithLimiter(&globalshell))
 
 	err := r.SendPayload(&Item{})
```

Comment:
I'm not clear what a lot of these cases were actually _trying_ to verify, since they expect an error (or not), but were not very prescriptive about _which_ error. Without the 'endpoint limit' applied to the `Item`, many of these were failing, but for the **wrong** reason, and still passing the test